Makefile

```diff
@@ -315,11 +315,21 @@ ifeq (, $(shell which ccache))
 $(info $(yellow)Warning: 'ccache' not found, consider installing it for faster builds$(term-reset))
 endif
 
+# Warn maintainers if bazel is not found.
+#
+# TODO(irfansharif): Assert here instead, on a fixed version of bazel (3.7.0). Something like:
+#
+#   $(error $(yellow)'bazel' not found (`brew install bazel` for macs)$(term-reset))
+ifeq (, $(shell which bazel))
+$(info $(yellow)Warning: 'bazel' not found (`brew install bazel` for macs)$(term-reset))
```

Comment:
Just passing by! Consider recommending the usage of github.com/bazelbuild/bazelisk and a `.bazelversion` file to ensure your contributors are using the same version.

Great to see CockroachDB using Bazel!

Reply:
Thanks for the suggestion! Do keep passing on by for more, we're still learning how bazel works so it's all the more welcome, @jin.